src/profile-logic/old-cleopatra-profile-format.js

```diff
@@ -120,22 +120,10 @@ function _convertThread(
   const frameMap = new Map();
   const stackMap = new Map();
 
-  let lastSampleTime = 0;
```

Comment:
Do we need to touch this code? I'd prefer to leave it alone and not convert old profiles unless we have a good reason. This is risky for backwards compatibility.

Reply:
I'd like to defer this decision to @mstange. @mstange have you tried the patch with an old profile?

Reply:
I'll feel comfortable if this is covered by tests (I didn't actually dig into it to see how this affects things) although my preference would be to leave old migration code alone, and then just lose that information.

Reply:
The existing old cleopatra profile `old-cleopatra-profile.sps.json` contains a marker which has a stack, at `profile.profileJSON.threads[0].markers[45].data.stack`.

We already have a test that runs profile conversion + processing on that profile, so we could add a section to that test which finds the corresponding marker in the processed profile, gets a stringified stack for that marker, and then checks that it matches. Julien, would you like to do that?

Reply:
Yep I was planning to look at it, thanks to look it up for me :)